migrator/migrations/n_42_to_n_43_postgres_process_indicators/postgres/postgres_plugin.go

```diff
@@ -153,6 +151,11 @@ func (s *storeImpl) copyFromProcessIndicators(ctx context.Context, tx *postgres.
 			return marshalErr
 		}
 
+		if pgutils.NilOrUUID(obj.GetId()) == nil {
+			log.Warnf("Id is not a valid uuid -- %v", obj)
```

Comment:
Don't print a process indicator as it can potentially have secrets in the command line. Perhaps the same applies for secret below even though we don't take the data fields